packages/forklift-console-plugin/src/modules/Providers/utils/components/DetailsPage/DetailItem.tsx

```diff
@@ -27,37 +27,67 @@ import Pencil from '@patternfly/react-icons/dist/esm/icons/pencil-alt-icon';
  */
 export const DetailsItem: React.FC<DetailsItemProps> = ({
   title,
-  content,
   helpContent,
   showHelpIconNextToTitle,
   moreInfoLabel,
   moreInfoLink,
   crumbs,
+  content,
+  contents,
   onEdit,
+  onEdits,
 }) => {
   return (
     <DescriptionListGroup>
-      {helpContent ? (
-        <DescriptionTitleWithHelp
-          title={title}
-          helpContent={helpContent}
-          showHelpIconNextToTitle={showHelpIconNextToTitle}
-          moreInfoLabel={moreInfoLabel}
-          moreInfoLink={moreInfoLink}
-          crumbs={crumbs}
-        />
-      ) : (
-        <DescriptionTitle title={title} />
-      )}
-      {onEdit ? (
-        <EditableContentButton content={content} onEdit={onEdit} />
-      ) : (
-        <NonEditableContent content={content} />
-      )}
+      <DisplayTitle
+        title={title}
+        helpContent={helpContent}
+        showHelpIconNextToTitle={showHelpIconNextToTitle}
+        moreInfoLabel={moreInfoLabel}
+        moreInfoLink={moreInfoLink}
+        crumbs={crumbs}
+      />
+      <DescriptionListDescription>
+        <ContentField content={content} onEdit={onEdit} />
+
+        {contents?.map((value, index) => (
+          <ContentField
+            key={'content-field-' + index}
+            content={value}
+            onEdit={onEdits ? onEdits[index] : null}
+          />
+        ))}
```

Comment:
see: https://github.com/kubev2v/forklift-console-plugin/pull/1135/files#r1597880185

convert content to array:
`contents = ensureArray(contnet)`

and then use it as array
```suggestion
        {contents?.map((value, index) => (
          <ContentField
            key={'content-field-' + index}
            content={value}
            onEdit={onEdits ? onEdits[index] : null}
          />
        ))}
```